docroot/modules/humsci/hs_layouts/patterns/date-stacked-vertical-card/date-stacked-vertical-card.html.twig

```diff
@@ -43,7 +43,7 @@
 
     {% if title %}
       <div class="hb-card__title">
-        {% if title|render matches '/<h\\d>/' %}
+        {% if title|render matches '/<\\/h\\d>/' %}
```

Comment:
I'm curious what didn't the original regex catch that you need to change?

Reply:
@pookmish The original regex only matched heading tags without arguments, e.g. `<h4>`. There are some sites where the headings have classes and the match was failing in those cases.

Reply:
Ahh! because the closing `>` in the regex. so `/<h\\d/` probably would work just as well, unless somewhere there was a tag like `<h123foobar>` I suppose.
Thanks for the info.